phenopacketlab-spring-boot-autoconfigure/src/main/java/org/monarchinitiative/phenopacketlab/autoconfigure/PhenopacketLabDataResolver.java

```diff
@@ -13,19 +15,59 @@ public class PhenopacketLabDataResolver {
     public PhenopacketLabDataResolver(Path phenopacketLabDataDirectory) throws MissingPhenopacketLabResourceException {
         this.phenopacketLabDataDirectory = phenopacketLabDataDirectory;
 
-        List<Path> paths = List.of(hpoJsonPath(), hpoAnnotationPath());
+        List<String> errors = new LinkedList<>();
+        List<Path> paths = List.of(efoJsonPath(), genoJsonPath(), hgncCompleteSetPath(), hpJsonPath(), mondoJsonPath(),
+                hpoAnnotationPath(), soJsonPath(), uberonJsonPath(), ncitJsonPath(), gssoJsonPath());
         for (Path path : paths) {
             if (!(Files.isRegularFile(path) && Files.isReadable(path))) {
-                throw new MissingPhenopacketLabResourceException(String.format("The file `%s` is missing in the data directory", path.toFile().getName()));
+                errors.add(path.toFile().getName());
             }
         }
+
+        if (!errors.isEmpty()) {
+            String missing = errors.stream().collect(Collectors.joining("', '", "'", "'"));
+            String message = String.format("The following files are missing in the data directory: %s.", missing);
+            throw new MissingPhenopacketLabResourceException(message);
+        }
     }
 
-    public Path hpoJsonPath() {
+    public Path hpJsonPath() {
         return phenopacketLabDataDirectory.resolve("hp.json");
     }
 
     public Path hpoAnnotationPath() {
         return phenopacketLabDataDirectory.resolve("phenotype.hpoa");
```

Comment:
Do we ever want to reconfigure all these names? Better in environment vars and reconfigurable on deployment?